themes/tdr/login/login-config-totp.ftl

```diff
@@ -0,0 +1,78 @@
+<#import "template.ftl" as layout>
```

Comment:
Was checking adding a code manually, instead of scanning the QR code, for MS authenticator, and not sure it works. MS autthenticator seems to require a numerical code instead of a mixture of alpha/numerical. It might worth double checking this, if this is the case, this probably needs raising, to see if we want to remove this option.

Reply:
I've tried this with the Microsoft Authenticator app on my phone and it's working ok I think,